<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USERLOGINClass</class>
 <widget class="QMainWindow" name="USERLOGINClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USERLOGIN</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-220</x>
      <y>-20</y>
      <width>1111</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginContainer" native="true">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>291</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mini Wasalny</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="USERLOGIN.qrc"/>
 </resources>
 <connections/>
</ui>
